src/screens/Ootd/OotdScreen.tsx

```
@@ -54,129 +56,213 @@ export function OotdScreen({ navigation }: Props) {
+
   return (
     <>
       <OotdTop />
-      <View style={styles.container}>
-        <View style={styles.topRow}>
-          <Pressable onPress={() => setIsStarred((p) => !p)} style={styles.iconBtn}>
-            {isStarred ? <StarIcon width={20} height={20} /> : <EmptyStarIcon width={20} height={20} />}
-          </Pressable>
+      <Pressable style={{ flex: 1 }} onPress={() => isDeleteMode && exitDeleteMode()}>
+        <View style={styles.container} pointerEvents="box-none">
+            <View style={styles.topRow} pointerEvents="box-none">
```

ESLint

[error] Delete `··`

(prettier/prettier)